src/bp-search/classes/class-bp-search.php

```diff
@@ -522,7 +522,7 @@ public function do_search( $args = '' ) {
 					return;
 				}
 
-				$pre_search_query = implode( ' UNION ', $sql_queries ) . ' ORDER BY relevance, type DESC, entry_date DESC ';
+				$pre_search_query = implode( ' UNION ', $sql_queries ) . ' ORDER BY relevance DESC, type DESC, entry_date DESC ';
```

Comment:
@ngolwalkar This change is not needed. We just need to update the order of the member results on the All results page.
Now you can see the order was changed based on the side menu: http://prntscr.com/1dsla0z

Reply:
@KartikSuthar Do you think we should fix the search filter instead of the search result? As per the query, we are asking for the most relevant result and when I dump the result it shows the relevance for the member as 1 while relevance for the reply as 0. So the member module has the most relevant result so the sequence seems to be good for the result while the search filter has the wrong sequence. Please let me know your views.